src/lib/backpack-api.js

```diff
@@ -62,7 +62,7 @@ const deleteBackpackObject = ({
 }) => new Promise((resolve, reject) => {
     xhr({
         method: 'DELETE',
-        uri: `${host}${username}/${id}`,
+        uri: `${host}/${username}/${id}`,
```

Comment:
Again, these incorrect paths were found while testing host config including the pre-existing configuration of the backpack host.